Statistics.py

```diff
@@ -0,0 +1,102 @@
+# Statistics.py
+# Computes statistics and creates cost histogram
+# Ran Libeskind-Hadas, June 2020
+
+# Performs Monte Carlo simulation to compute empirical p-value and histogram
+# of solution costs.
+# https://www.ncbi.nlm.nih.gov/pmc/articles/PMC379178/
+
+from clumpr import DTLReconGraph
+import random
+import matplotlib
+matplotlib.use('tkagg') # need this so plt.show() works
+import matplotlib.pyplot as plt
+
+from typing import List, Dict, Tuple
+
+def trials(host_tree: dict, parasite_tree: dict, phi: dict, dup_cost: float, transfer_cost: float, loss_cost: float, num_trials: int) -> list:
```

Comment:
Do you want other files to call this function `trails` and `create_random_phi`? Or do you want it to be internal? You can tell the users of this file that some functions are internal by naming functions with a leading underscore `_trails` and `_create_random_phi` or you can create an `__all__` variable at the top of the module which lists all functions you intend to export. The functions with leading underscore or not in all variable will not be suggested to users by autocomplete tools such as vscode python extension and will not be documented by document generation tools such as sphinx.